src/wasm/mod.rs

```diff
@@ -334,6 +334,29 @@ impl PairedBitBox {
         Ok(psbt.to_string())
     }
 
+    #[wasm_bindgen(js_name = btcSignMessage)]
+    pub async fn btc_sign_message(
+        &self,
+        coin: types::TsBtcCoin,
+        script_config: types::TsBtcScriptConfigWithKeypath,
+        msg: &[u8],
+    ) -> Result<types::TsBtcSignMessageSignature, JavascriptError> {
+        let signature = self
+            .0
+            .btc_sign_message(coin.try_into()?, script_config.try_into()?, msg)
+            .await?;
+
+        Ok(
+            serde_wasm_bindgen::to_value(&crate::btc::SignMessageSignature {
```

Comment:
I think you can just do `serde_wasm_bindgen::to_value(signature)` right? no need to create a copy of the struct instance.

Reply:
Now that I use the actual struct from `btc`, yes I can I suppose.